src/main/kotlin/stringLettersCalculator/StringNumbers.kt

```diff
@@ -0,0 +1,21 @@
+package stringLettersCalculator
+
+class StringNumbers(
+    private val stringNumbers: List<StringNumber> = listOf()
+) {
+
+    fun numbersToInt(): List<Int> = stringNumbers.map {
+        it.toInt()
+    }
+
+    fun sumNumbers(): Int = stringNumbers.map { it.toInt() }
+        .sumOf { it }
+
+    companion object {
+        fun from(numberStrings: List<String>) = StringNumbers(
+            List(numberStrings.size) {
+                StringNumber(numberStrings[it])
+            }
+        )
```

Comment:
아래 처럼 메서드 체이닝을 활용해볼 수 있겠어요 :)
```suggestion
        fun from(numberStrings: List<String>): StringNumbers {
            return numberStrings.map { StringNumber(it) }
                    .let { StringNumbers(it) }
        )
```

Reply:
오! 감사합니다! 사실 저 표현법도 자동차경주에서 처음 봤을땐 신세계같았거든요! 더 새로운게 있었네요!